components/button/floating-buttons.js

```diff
@@ -0,0 +1,236 @@
+import 'lit-media-query/lit-media-query.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { ButtonMixin } from './button-mixin.js';
+import { buttonStyles } from './button-styles.js';
+import { labelStyles } from '../typography/styles.js';
+
+class FloatingButtons extends ButtonMixin(LitElement) {
+
+	constructor() {
```

Comment:
we have been mostly ordering the component code `properties`, `styles`, `constructor`, updated/callback functions, `render`, other public functions, other private functions